src/ParallelAwareConfigInterface.php

```diff
@@ -19,9 +19,9 @@
 /**
  * @author Greg Korba <greg@codito.dev>
  *
- * @TODO 4.0 Include parallel runner config in main config
+ * @TODO 4.0 Include parallel runner config in main ConfigInterface
  */
-interface ParallelRunnerConfigInterface extends ConfigInterface
+interface ParallelAwareConfigInterface extends ConfigInterface
```

Comment:
`ParallelAwareConfigInterface` is `ConfigInsterface` that is aware of `Parallel` aspect.
It's Runner config, but `ConfigInterface` is not exposing the `Runner` in name (and I think it should not)

for `ParallelConfigurationInterface` , that was my first choice, and then I noticed `src/Runner/Parallel/ParallelConfig.php:final class ParallelConfig` (which is totally different thing)

If it convince you, let's move on with it. If not, maybe `ConfigV4Interface` ?

Reply:
For me `ParallelRunnerConfigInterface` is just OK, as it explicitly says that it's about configuration related to parallel _runner_ - it's important IMHO, as it shows that this config directly affects how the tool is _run_. With "aware" approach it is not so clear (for me).

But since it's a temporary interface (to be removed in v4), it does not make sense to argue about it for too long 🙂. Please just let me know which one we should use.

Reply:
I'll use `ParallelAwareConfigInterface`.